scripts/proxy/browse_file_operations.py

```
@@ -0,0 +1,1167 @@
+                    current_tree[entry.name] = FileOperation(
+                        path=str(entry), was_read=False, was_written=False
+                    )
+
+    walk_dir(root, tree, 0)
+
+    # Add any touched files that weren't found in the tree (outside root or deleted)
+    missing_files = []
+    for orig_path, op in file_ops.items():
+        try:
+            resolved = str(Path(orig_path).resolve())
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)